stage1/rootfs/usr_from_src/Makefile

```diff
@@ -31,10 +31,12 @@ systemd.done: Makefile systemd.src
 
 # TODO(vc): it may make more sense to have the systemd source be a git submodule?
 systemd.src: Makefile patches/*
+	if [ -z "$(RKT_STAGE1_SYSTEMD_SRC)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_SRC undefined"; exit 1; fi
+	if [ -z "$(RKT_STAGE1_SYSTEMD_VER)" ]; then echo "Error: RKT_STAGE1_SYSTEMD_VER undefined"; exit 1; fi
 	{ [ ! -e systemd ] || rm -Rf systemd; }
 	mkdir systemd
-	git clone --branch v215 --depth 1 git://anongit.freedesktop.org/systemd/systemd
-	cd systemd && for p in ../patches/*; do patch -p1 <"$$p"; done
+	git clone --branch $(RKT_STAGE1_SYSTEMD_VER) --depth 1 $(RKT_STAGE1_SYSTEMD_SRC)
+	if [ -d patches/$(RKT_STAGE1_SYSTEMD_VER) ]; then cd systemd && for p in ../patches/$(RKT_STAGE1_SYSTEMD_VER)/*.patch; do patch -p1 <"$$p"; done; fi
```

Comment:
Ok, I will do that. I need to add at the top of the Makefile:
```
# make needs to use bash for nullglob
SHELL := /bin/bash
```

and I added some `\` or `; \` when needed.